src/Twig/Extension/IconExtension.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Twig\Extension;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+class IconExtension extends AbstractExtension
+{
+
+    public function getFilters(): array
+    {
+        return array(
+            new TwigFilter('parse_icon', array($this, 'parseIcon')),
+        );
+    }
+
+    public function parseIcon(string $icon, bool $inline = false): string
```

Comment:
Not sure `$inline` is clear.

I would call this `$onlyClass` or something like this.

Reply:
When I see the usage, I would say that a string `extraClasses` would be better.

It could be used this way: 
```
{{ icon|parseIcon('sonata-ba-list-field-header-label-icon') }}
```